pkg/apis/core/v1/defaults.go

```diff
@@ -60,10 +60,7 @@ func SetDefaults_ReplicationController(obj *v1.ReplicationController) {
 			obj.Labels = labels
 		}
 	}
-	if obj.Spec.Replicas == nil {
-		obj.Spec.Replicas = new(int32)
-		*obj.Spec.Replicas = 1
-	}
+	// obj.Spec.Replicas is defaulted declaratively
```

Comment:
apologies for the dense question, but could I get a link?  I don't see it in https://github.com/kubernetes/kubernetes/blob/85488b5f10bc6128dab69e4bdd94cec2c005c7f5/pkg/apis/core/v1/zz_generated.defaults.go#L879

Reply:
From the [below comment](https://github.com/kubernetes/kubernetes/pull/130725/files#r1991378264), looks like you found it?